ecommerce/products/management/commands/create_fixtures.py

```diff
@@ -0,0 +1,28 @@
+import random
+
+from django.core.management.base import BaseCommand
+from products.utils.fixtures import word_name, fake_description, fake_price,\
+    fake_date, \
+    choice_from_choice_class, fake_product, fake_product_item
```

Comment:
Couple things,

(1) If you have too many imports, a neater way is to just do `from products.utils import fixtures`, then you can do `fixtures.fake_product` in the code.

(2) Most of these imports aren't used in this script. You only need `fake_product` and `fake_product_item`